config/crd/bases/atlas.mongodb.com_atlasprojects.yaml

```diff
@@ -887,6 +887,17 @@ spec:
                       type: string
                   type: object
                 type: array
+              regionUsageRestrictions:
+                default: NONE
+                description: RegionUsageRestrictions designate the project's AWS region
+                  when using Atlas fo Government. This parameter should not be used
+                  with commercial Atlas In Atlas for Government, not setting this
+                  field (defaulting to NONE) means the project is restricted to COMMERCIAL_FEDRAMP_REGIONS_ONLY
```

Comment:
> RegionUsageRestrictions designate the project's AWS region when using Atlas fo Government.

It does not designate a particular AWS region, right? I guess it limits the choice of AWS regions.

> This parameter should not be used with commercial Atlas In Atlas for Government

Strange, one would assume setting this field to something other than NONE would be precisely when the Atlas cloud setup is government related.

> not setting this field (defaulting to NONE) means the project is restricted to COMMERCIAL_FEDRAMP_REGIONS_ONLY

Are we saying here NONE becomes same as COMMERCIAL_FEDRAMP_REGIONS_ONLY??

For me the bare enum seems intuitive, but after reading the description I believe my intuitions might be all wrong.


Can you elaborate more about this setting?

Reply:
I prefer to make it clear in the description because no setting means `COMMERCIAL_FEDRAMP_REGIONS_ONLY`, setting it to `NONE` also means `COMMERCIAL_FEDRAMP_REGIONS_ONLY`.